src/index.ts

```diff
@@ -883,6 +883,27 @@ export class Bigtable {
     }
   }
 
+  /**
+   * Terminate grpc channels and close all the clients.
+   */
+  close(): Promise<void> | Promise<void[]> {
```

Comment:
The only thing I would consider here is that since `Promise.all(combined)` returns an array then we should replace `Promise.resolve()` with `Promise.resolve([])` so that the output is always the same type. Maybe that's a bad idea though and for some reason we want the types to be different. Not sure.